src/NodaTime/AnnualDate.cs

```diff
@@ -28,7 +32,8 @@ namespace NodaTime
     /// </para>
     /// </remarks>
     [TypeConverter(typeof(AnnualDateTypeConverter))]
-    public readonly struct AnnualDate : IEquatable<AnnualDate>, IComparable<AnnualDate>, IComparable, IFormattable
+    [XmlSchemaProvider(nameof(GetSchema))]
+    public readonly struct AnnualDate : IEquatable<AnnualDate>, IComparable<AnnualDate>, IComparable, IFormattable, IXmlSerializable
```

Comment:
I would rather not add XML serialization for AnnualDate in the same PR as support for XML schemas. I don't mind which order we do them in, but I'd really like to do one at a time.

Reply:
Done and you already merged #1502, thanks!